configs/open5gs/sgwc.yaml.in

```diff
@@ -82,6 +82,11 @@ sgwc:
       - addr: 127.0.0.3
     pfcp:
       - addr: 127.0.0.3
+    cdr:
+      enabled: True
+      file_capture_period_sec: 1800
+      reporting_period_sec: 10
+      sgw_name: SGW-01
```

Comment:
Do we need to add `log_dir` parameter here? @Ayrdim

Reply:
Oops forgot this .in file example, yeah its been added